test/application_system_test_case.rb

```diff
@@ -1,5 +0,0 @@
-require "test_helper"
-
-class ApplicationSystemTestCase < ActionDispatch::SystemTestCase
-  driven_by :selenium, using: :chrome, screen_size: [1400, 1400]
```

Comment:
never knew rails do selenium end to end testing

Reply:
We will be doing specs with RSpec and we can add selenium there.